tests/autobahn/server/docker-compose.yml

```diff
@@ -14,5 +14,7 @@ services:
     command: ["wstest", "--mode", "fuzzingclient", "--spec", "/config/fuzzingclient.json"]
 
   aiohttp:
```

Comment:
Does aiohttp itself really need to run in container? Maybe just autobahn would be enough.

Reply:
Yes no need to run aiohttp in a container.